Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/update/DccWalletInfoUpdateTrigger.kt

```diff
@@ -35,8 +34,6 @@ class DccWalletInfoUpdateTrigger @Inject constructor(
     init {
         appScope.launch {
             personCertificateProvider.personCertificates
-                .drop(1) // Drop first value on App start that causes unnecessary calculation
```

Comment:
Not entirely unnecessary. Yes, removing this line will cause a wallet recalculation on each app start, but will prevent any failed recalculations from closing the app too early. This was the most non-intrussive fix I could think of.